### PR TITLE
Position creepy face 

### LINKED ISSUE #5412: PNI - homepage - creep-o-face-o-location-o
**Tasks**
- [x] update the color of the speech bubble 
- [x] add in the styling tweaks to match new comps
- [x] update the location of the creep-o-meter content when search is active (see description below)
  - [x] mobile
  - [x] desktop
- [x] update the location of the creep-o-meter during scroll (see description below)
  - [x] mobile
  - [x] desktop
- [x] update initial creep-o-meter content to contain the intro text

**Desktop**
- [Creepy face appears center between search and checkbox with intro speech bubble](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=103e293c-16c0-48af-bac2-dc7d7545e5ea&mode=design)
- [When search is selected, search bar expands and creepy face relocates to bottom right corner ](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=6619f77a-a5e3-4019-b224-8adf02478629&mode=design) _(if search bar is closed creepy is center)_
- [If user is typing and creepy isn't determined yet, we can use a neutral smiley face with no speech bubble](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=a05bb884-48f2-47d3-b0ea-4ca413cd4212&mode=design)
- Faces change depending on how creepy the products are in viewport _(from not creepy to super creepy, see below)_
- Creepy face appears sticky up until last row of products (just after it so it doesn't block product thumbnail)

**Mobile**
- [Creepy face only appears center during initial presentation since search bar is open by default on mobile](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=3f690fff-c322-4f11-8505-04aee49fea04&mode=design)
- [As user scroll past intro creepy face, it relocates to bottom right corner of viewport](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=8d1fc7b3-fad0-4bdd-9815-6ef8ee18e5c8&mode=design)
- Creepy face appears sticky up until last row of products (just after it so it doesn't block product thumbnail)

---

###  Design -> Dev Handoff Checklist
(Feel free to remove items that are not applicable.)

- [x] Design has been finalized
- [x] Link to related ticket: #5314
- [x] Link to Abstract (make sure it can be viewed by anyone)
- [x] Design changes been merged into the master Sketch file
  - Merged into master Sketch file via Abstract. (_Master > Buyers Guide.sketch > [Home Page Updates (Q4 2020](https://share.goabstract.com/07a9aca5-3903-4f00-828a-6ac8bebb5be3?sha=e49d77cc9da6a70e8c412cd985e28138a3091cd5))_).
- [x] Desktop and mobile mockups are included
  - [[Desktop] Homepage - Index](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=103e293c-16c0-48af-bac2-dc7d7545e5ea&mode=design) _(initial presentation)_
  - [[Desktop] Homepage - Scroll ](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=44c49575-bd16-462d-a712-b76353238f92&mode=design)  _(sticky by bottom right corner of view port on scroll)_
  - [[Mobile] Homepage](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=3f690fff-c322-4f11-8505-04aee49fea04&mode=design)  _(initial presentation)_
  - [[Mobile] Homepage -  Scroll](https://share.goabstract.com/e9fd07c9-605e-4ca9-b99c-7a3686de51cb?collectionLayerId=8d1fc7b3-fad0-4bdd-9815-6ef8ee18e5c8&mode=design) (_sticky by bottom right corner of view port on scroll)_
- [x] Name of the Sketch file (and specific page/section to look at, if applies)
- [x] Assets are included. Link: https://drive.google.com/open?id=1AtHaj5YuXTQh03uDbz2JwZ6dPLxM3UiO&authuser=sabrina%40mozillafoundation.org&usp=drive_fs 
  - Note: bubbles preferably done purely in HTML / CSS so it is responsive
- [x] Other states for elements are designed: various face changes on scroll and all assets exported
image

### PR DESCRIPTION
**Pull request by KalobTaulien**
Partially Closes #5412 

**Note** I also made the face disappear when there are no search results. This requires UI testing. 

Live URL: https://foundation-s-5412-creep-x8rvk6.herokuapp.com/privacynotincluded

### CODE REVIEW

**Comment by Pomax** [comment]
Scroll through this, it feels like we should probably make the creep-o-meter disappear if it goes past the products, or restrict its position somehow?

E.g. this is a little odd:

image

**Review by sabrinang — CHANGES_REQUESTED** [review]
In issue https://github.com/mozilla/foundation.mozilla.org/issues/5412#issue-718353698 under desktop/mobile there is a bullet point: 

- Creepy face appears sticky up until last row of products (just after it so it doesn't block product thumbnail)

I took screenshot and notes of approximately where the face can be sticky up until:
Screen Shot 2020-11-02 at 1 55 45 PM

File: source/js/buyers-guide/search.js [hunk 1]
```diff
@@ -82,6 +83,16 @@ const SearchFilter = {
...
+    // When searching, check to see how many products are still visible
+    // If there areno visible products, there are "no search results"
+    // And when there are no searchr results, do not show the creepo-meter-face
+    if (document.querySelectorAll(".product-box:not(.d-none)").length) {
+      // If there are search results, show the creepo-meter-face
+      CREEPINESS_FACE.classList.remove("d-none");
+    } else {
+      // If there are no search results, hide the creepo-meter-face
+      CREEPINESS_FACE.classList.add("d-none");
+    }
```
Comment (mmmavis) [inline source/js/buyers-guide/search.js:95]: Let move this to a function so we can call it in `clearText()` as well.

**Review by mmmavis — CHANGES_REQUESTED** [review]
Left 2 comments + the hopefully-its-just-me creepy face disappearing issue that I mentioned on Slack.

**Comment by KalobTaulien** [comment]
Closing this PR as it's been manually merged into #5558

**Pull request closed without merging**